introduction/examples/thompson/erastothenes.py

```diff
@@ -0,0 +1,33 @@
+# Function returns a list of prime numbers from 2 to n
+def eratosthenes(n):
+    if isinstance(n, int):
```

Comment:
Lepiej jest zastosować `if not isinstance...` i od razu wyjść z funkcji. Dzięki czemu nie tworzymy niepotrzebnych wcięć (czytelność)